messages/events/account.go

```diff
@@ -35,3 +39,47 @@ type AccountDebitDeclined struct {
 	Amount          int64
 	Reason          messages.DebitFailureReason
 }
+
+// MessageDescription returns a human-readable description of the message.
+func (m AccountOpened) MessageDescription() string {
+	return fmt.Sprintf(
+		"account %s %s opened for customer %s",
```

Comment:
```suggestion
		"opened account %s %s for customer %s",
```

(just to mirror the command's description)